### DISCUSSION TITLE
ec2 instance

### DISCUSSION THREAD

**Discussion by nandini965** [learndevopsonline/learndevopsonline#782] [Q&A] [answered]
if iam using 't3 small' , new  instance are continously getting launched even though am terminating them
![Uploading Screenshot (27).png…]()

**Comment by chashish9** [comment, marked as answer]
We need to cancel the spot request also to ternimate the instance . We can only stop the instance when  we use the spot request

**Comment by nandini965** [comment]
okay thanq
